.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Various reformattings
+53ee471644fa767da243114abe51efeb705618ae
```

Comment:
I'm OK with pushing this. But I have no idea what this is :)

Reply:
It makes it easier to use `git blame`: https://moxio.com/blog/ignoring-bulk-change-commits-with-git-blame/